pkg/instancegroups/instancegroups.go

```diff
@@ -43,6 +43,26 @@ import (
 
 const rollingUpdateTaintKey = "kops.k8s.io/scheduled-for-update"
 
+// ValidationTimeoutError represents an error that occurs when
+// the cluster fails to validate within the designated timeout.
+type ValidationTimeoutError struct {
+	Operation string
+	Err       error
```

Comment:
There should be an `Unwrap` method on this struct as well that returns this value. I also think neither `err` nor `operation` should be exported.

```
func (v *ValidationTimeoutError) Unwrap() error { 
  return v.Err 
}
```